package.json

```diff
@@ -9,7 +9,8 @@
   ],
   "scripts": {
     "build": "tsc",
-    "test": "./node_modules/.bin/nyc ./node_modules/.bin/mocha -r ts-node/register test/*.ts"
+    "auth": "./node_modules/.bin/ts-node ./test/http-server-oauth-authorize.ts",
```

Comment:
To get OAuth2 token: npm run auth. As far as I understand, now fortnox only allows OAuth2 for "new" apps